data-prepper-core/src/integrationTest/resources/raw-span-e2e-pipeline.yml

```diff
@@ -0,0 +1,25 @@
+entry-pipeline:
+  source:
+    otel_trace_source:
+      ssl: false
+  sink:
+    - pipeline:
+        name: "raw-pipeline"
```

Comment:
Comment for later - eventually we might consider integ tests per component/feature instead of adding more to the existing. Calling this out because at first glance this pipeline config might look "incomplete" (no peer forwarder), but that is by design to test the traceGroupPrepper.

End state would be tests for:
1. Basic test- Single DP
2. Peer Forwarder plugin test - Two DP with forwarding enabled 
3.  TraceGroup plugin test - Two DP without forwarding
etc.

Will be easier to isolate breaking changes based on the specific test failing. I'll create a backlog item if we think this is a good idea.

Reply:
Sounds good to me. Might even add one with Amazon Elasticseach Service backend.

Reply:
Created #492